tests/phpunit/tests/functions/getCalendar.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Tests for the get_calendar() function.
+ *
+ * @since 6.8.0
+ *
+ * @group functions
+ * @group calendar
+ *
+ * @covers ::get_calendar
+ */
+class Tests_Get_Calendar extends WP_UnitTestCase {
+
+	/**
+	 * Array of post IDs.
+	 *
+	 * @var int[]
+	 */
+	protected static $post_ids = array();
+
+	/**
+	 * Set up before class.
+	 *
+	 * @param WP_UnitTest_Factory $factory Test factory.
+	 */
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$post_ids = $factory->post->create_many(
+			3,
+			array(
+				'post_date' => '2025-02-01 12:00:00',
+			)
+		);
+	}
+
+	/**
+	 * Test that get_calendar() displays output when display is true.
+	 *
+	 * @ticket 34093
+	 */
+	public function test_get_calendar_display() {
+		$expected = '<table id="wp-calendar"';
+		ob_start();
+		get_calendar( array( 'display' => true ) );
+		$actual = ob_get_clean();
```

Comment:
By the way, the tests library has a nice util function for this called `get_echo()`

`get_echo( 'get_calendar', array( array( 'display' => true ) ) )`